packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -126,6 +127,7 @@
 		"mocha": "^10.2.0",
 		"nyc": "^15.1.0",
 		"prettier": "~2.6.2",
+		"process": "^0.11.10",
```

Comment:
Seems weird that this was working before without this dependency.

Reply:
This package is used to generate browser extension. We run application test under /devtools-example

Reply:
@tylerbutler we didn't use the `process` variable at all before in this package. In fact, is the dep even necessary? We're not really going to use it, it just serves as an "anchor" for `dotenv-webpack` to do its inline replacements... If `dotenv-webpack` wants it as a peer dep, let's keep it, but otherwise I'd say we should remove it.

Reply:
I think we need to access the environment variables by `process.env.DEVTOOL_TELEMETRY_TOKEN`

Reply:
That's kind of the point... we _don't_ access them like that. `dotenv-webpack` replaces that code with an inlined value so during runtime the text `process.env` is not even there.

Reply:
Gotcha. Seems like dotenv-webpack will replace the DEVTOOLS_TELEMETRY_TOKEN during build.